src/display-buffer.coffee

```diff
@@ -465,6 +465,10 @@ class DisplayBuffer extends Model
 
     folds
 
+  # Returns all the folds that contain the given row range.
+  foldsContainingBufferRowRange: (startRow, endRow) ->
```

Comment:
Yeah, I thought to this as well but this _seems_ inconsistent with the method below (`::foldsContainingBufferRow`) which performs a similar query. What do you think?